gherkin/elixir/lib/gherkin/dialect.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Gherkin.Dialect do
+  @type keywords :: [String.t(), ...]
+  @type t :: pid
+
+  @spec and_keywords(t) :: keywords
+  def and_keywords(dialect), do: fetch(dialect, "and")
+
+  @spec background_keywords(t) :: keywords
+  def background_keywords(dialect), do: fetch(dialect, "background")
+
+  @spec but_keywords(t) :: keywords
+  def but_keywords(dialect), do: fetch(dialect, "but")
+
+  @spec examples_keywords(t) :: keywords
+  def examples_keywords(dialect), do: fetch(dialect, "examples")
+
+  @spec feature_keywords(t) :: keywords
+  def feature_keywords(dialect), do: fetch(dialect, "feature")
+
+  @spec given_keywords(t) :: keywords
+  def given_keywords(dialect), do: fetch(dialect, "given")
+
+  @spec scenario_keywords(t) :: keywords
+  def scenario_keywords(dialect), do: fetch(dialect, "scenario")
+
+  @spec scenario_outline_keywords(t) :: keywords
+  def scenario_outline_keywords(dialect), do: fetch(dialect, "scenarioOutline")
+
+  dialects =
+    :gherkin
+    |> :code.priv_dir()
+    |> Path.join("gherkin-languages.json")
```

Comment:
May be useful to use https://hexdocs.pm/elixir/Module.html#module-external_resource regarding this line, nbd though. Would cause this file to correctly recompile if the `.json` file changed.

Reply:
Good call!